armada/cli/apply.py

```diff
@@ -39,6 +35,8 @@ def get_parser(self, prog_name):
                             help='Armada yaml file')
         parser.add_argument('--dry-run', action='store_true',
                             default=False, help='Run charts with dry run')
+        parser.add_argument('--trace', action='store_true',
```

Comment:
Since --debug is reserved by Cliff, @gardlt and I decided to go with --trace. I do not have a preference as to whether it is --debug-logging or --trace. 

@gardlt would you be opposed to changing it?